engine/rpcserver.go

```diff
@@ -1658,6 +1659,16 @@ func (s *RPCServer) GetHistoricCandles(_ context.Context, req *gctrpc.GetHistori
 			Volume: candles.Candles[i].Volume,
```

Comment:
Kinda sorta out of scope nit that I can't place anywhere: Your GCTCLI commands  GetCandles and GetCandlesExtended ignore what currencies or asset types are disabled. It will just return data if it matches

Reply:
Noticed this morning have a fix that is coming that validates supported & enabled before execution on both